OrbitCore/CMakeLists.txt

```diff
@@ -213,6 +213,7 @@ target_sources(OrbitCoreTests PRIVATE
     RingBufferTest.cpp
     StringManagerTest.cpp
     LinuxTracingSessionTests.cpp
+    PathTest.cpp
```

Comment:
I think you're missing a file in the commit.

Reply:
right - fixed.